### ISSUE TITLE
Exploit completed, but no session was created

### ISSUE THREAD

**Issue by Git8888** [NAXG/cve_2019_0708_bluekeep_rce#3]
![image](https://user-images.githubusercontent.com/37396317/64484717-f320c480-d248-11e9-92de-6c0ce1bfd00c.png)

Exploit failed [disconnected]: Errno::ECONNRESET Connection reset by peer

**Comment by dcliss** [comment]
firewalls

**Comment by S1lkys** [comment]
I get the same error with disabled firewalls

**Comment by Git8888** [comment]
No, in fact, I turned off the firewall, he still has this problem.
